lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE IF NOT EXISTS emailBounces (
+  email VARCHAR(255) NOT NULL,
```

Comment:
I wonder if we should consider a bit of db normalization here.  We've got "change email" on our plate for Q1, and with it the possibility of having multiple emails per account.  Would it make sense to keep them in an "emails" table  with integer primary key, like:

```
+------+------------------+-----+
|  id  |  email           | ... |    
+------+------------------+-----+
|  123 | test@example.com | ... |
|  124 | user@gmail.com   | ... |
|  ... | ...              | ... |
+------+------------------+-----+
```

And then have other tables like this `emailBounces` one, refer to them by id.  It would save duplicating a big varchar column multiple times, at the cost of a bit of query complexity.